src/components/sections/senses_section.tsx

```diff
@@ -1,14 +1,136 @@
 import { Link } from "@/i18n/routing"
+import { useTranslations } from "next-intl"
+import Image from "next/image";
 
+const senses = [
+  {
+    name: "see",
+    imagesLeft: ["/example.jpg", "/example.jpg"],
+    imagesRight: [
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+    ]
+  },
+  {
+    name: "hear",
+    imagesLeft: ["/example.jpg", "/example.jpg"],
+    imagesRight: [
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+    ]
+  },
+  {
+    name: "smell",
+    imagesLeft: ["/example.jpg", "/example.jpg"],
+    imagesRight: [
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+    ]
+  },
+  {
+    name: "taste",
+    imagesLeft: ["/example.jpg", "/example.jpg"],
+    imagesRight: [
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+    ]
+  },
+  {
+    name: "touch",
+    imagesLeft: ["/example.jpg", "/example.jpg"],
+    imagesRight: [
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+      "/example.jpg",
+    ]
+  },
+];
 export default function SensesSection() {
+  const t = useTranslations("TextSwitcher");
   return (
-    <div className="">
-      <h1 className="text-4xl font-fancy" id="senses">
-        Senses
-      </h1>
-      <div className="flex flex-col gap-4">
-        <Link className="text-xl" href="/sight">
-          Sight
+    <div className="flex flex-col gap-12">
```

Comment:
Can you please move this to a separate file in `components/sections`? This is quite a lot of clutter for the home page